charts/openshift-metering/values.yaml

```diff
@@ -639,6 +647,10 @@ presto:
       terminationGracePeriodSeconds: 30
       tolerations: []
 
+      podDisruptionBudget:
+        minAvailable: 0
```

Comment:
I think something is off, either this number or the descriptive text, because all the text talks about minimum of one.

Reply:
That's probably a poor copy-paste job. We don't need to create a PDB resource for a resource that doesn't have any replicas, which is the default configuration for the Presto workers. That said, I don't think we would need to configure that kind of resource for non-coordinator resources anyways.